lib/determinator/retrieve/http_retriever.rb

```diff
@@ -0,0 +1,27 @@
+require 'faraday'
+
+module Determinator
+  module Retrieve
+
+    class HttpRetriever
+      def initialize(connection:)
```

Comment:
I tmight be good to have a default connection here — one with a sane UserAgent?

Reply:
I'll make a note of that in the documentation by way of example. You'd also expect a circuit breaker etc. It's a shame Routemaster::Cache is gone as it was the closest we had to a blessed httpclient.